fn generic_error(server) {
    print(server.timestamp);
}

fn throw_error() {
    throw "this is an error throwned by the user";
}